src/Symfony/Component/Notifier/Bridge/Mercure/MercureOptions.php

```diff
@@ -22,6 +22,22 @@ final class MercureOptions implements MessageOptionsInterface
 
     /**
      * @param string|string[]|null $topics
+     * @param array{
+     *     actions?: array,
```

Comment:
I would remove this one. https://developer.mozilla.org/en-US/docs/Web/API/Notification/Notification#actions says it can only be used for persistent notifications, so passing a value here would break things.